pinot-broker/src/test/resources/log4j2.xml

```diff
@@ -26,11 +26,12 @@
     </Console>
   </Appenders>
   <Loggers>
-    <Logger name="org.apache.pinot" level="warn" additivity="false">
+    <Root level="OFF"/>
```

Comment:
We usually need these logs to fix issues detected when a test fail. Although we can always turn it on at the time we are fixing the issue, do you know if there is any alternative way to do that? Maybe reading some property devs can set in their `~/.m2/settings.xml`?

Reply:
I assume this is only an issue for integration tests, where you need to have a pinot component is a specific state before running the tests. In such case the exception of the test is not enough; in such case the pinot component logging could help. 
It is possible to use system properties in log4j.xml files: https://logging.apache.org/log4j/2.x/manual/lookups.html
For me the main question is: should the logging be available when the tests fails, or is it good enough to rerun the test with adjusted settings/arguments?